bindings/python/test.py

```diff
@@ -50,6 +50,8 @@ def test_torrent_handle(self):
 		self.assertEqual(self.h.piece_priorities(), [4])
 
 		self.h.prioritize_files([0,1])
+		# workaround for asynchronous priority update
+		time.sleep(1)
```

Comment:
good find. I thought the priorities were still updated immediately, but then corrected when the disk job come back, is that not the case?

Reply:
I would prefer to restore the previous behavior here

Reply:
I've added this to get python tests passed. This workaround depends on `torrent::prioritize_files` implementation that is considered in this PR.

Reply:
maybe you're right. It does make sense to apply file priorities asynchronously, since the alternative is to (potentially) lie about the priorities. If we can't apply it, we shouldn't ever say it was applied.

The problem with making it async though is that there really should be an alert to indicate that it was applied.

Reply:
Yes, I like the idea of introducing new alert for this operation. This what I thought of, when posted an issue for improvement.